llvm/lib/Target/RISCV/RISCVProcessors.td

```diff
@@ -761,6 +761,37 @@ def SPACEMIT_X60 : RISCVProcessorModel<"spacemit-x60",
   let MImpID = 0x1000000049772200;
 }
 
+def SPACEMIT_X100 : RISCVProcessorModel<"spacemit-x100",
+                                        NoSchedModel,
+                                       !listconcat(RVA23S64Features,
```

Comment:
This needs 1 more space of indentation. The `!` should be in the same column as `NoSchedModel` above.

Reply:
Thanks, addressed and sorted the FeatureStd list in alphabetical order